sdk/include/reactos/mc/netmsgmsg.mc

```diff
@@ -3418,7 +3418,7 @@ Language=English
 The message alias could not be found on the network.
 .
 Language=Polish
-The message alias could not be found on the network.
+Nie można odnaleźć aliasu komunikatu w sieci lokalnej.
```

Comment:
Po prostu w sieci. To nie local network

Reply:
komunikaty zgodne z Windows :stuck_out_tongue_winking_eye: